analyzers/tests/SonarAnalyzer.UnitTest/Rules/BinaryOperationWithIdenticalExpressionsTest.cs

```diff
@@ -26,26 +26,39 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class BinaryOperationWithIdenticalExpressionsTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.BinaryOperationWithIdenticalExpressions>();
+        private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.BinaryOperationWithIdenticalExpressions>();
+
         [TestMethod]
         public void BinaryOperationWithIdenticalExpressions_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\BinaryOperationWithIdenticalExpressions.cs",
-                new CS.BinaryOperationWithIdenticalExpressions());
+            builderCS.AddPaths("BinaryOperationWithIdenticalExpressions.cs").Verify();
 
         [TestMethod]
         public void BinaryOperationWithIdenticalExpressions_TestProject_CS() =>
-            OldVerifier.VerifyNoIssueReported(@"TestCases\BinaryOperationWithIdenticalExpressions.cs",
-                new CS.BinaryOperationWithIdenticalExpressions(),
-                TestHelper.ProjectTypeReference(SonarAnalyzer.Helpers.ProjectType.Test));
+            builderCS.AddPaths("BinaryOperationWithIdenticalExpressions.cs")
+                .AddReferences(TestHelper.ProjectTypeReference(ProjectType.Test))
```

Comment:
Instead of the old way of adding test reference, you should use:
```suggestion
                .AddTestReference()
```